src/librustdoc/html/markdown.rs

```diff
@@ -1377,12 +1377,12 @@ fn init_id_map() -> FxHashMap<String, usize> {
     map.insert("copy-path".to_owned(), 1);
     map.insert("help".to_owned(), 1);
     map.insert("TOC".to_owned(), 1);
-    map.insert("render-detail".to_owned(), 1);
     // This is the list of IDs used by rustdoc sections.
     map.insert("fields".to_owned(), 1);
     map.insert("variants".to_owned(), 1);
     map.insert("implementors-list".to_owned(), 1);
     map.insert("synthetic-implementors-list".to_owned(), 1);
+    map.insert("foreign-impls".to_owned(), 1);
```

Comment:
What does "rustdoc templates" mean in this context? I assume it's not related to Tera templates since we only just now started migrating those.

Reply:
It refers to the generation of HTML in rustdoc. Not the best naming but I guess at the time it sounded like a good enough idea. :)

Reply:
Gotcha. Do you mind updating the comment to be more clear? Something like "This is the list of IDs used in HTML generated in Rust" and then "in JavaScript" respectively.

Reply:
Sure.